src/Draco.Compiler/Internal/Symbols/VariableSymbol.cs

```diff
@@ -14,4 +16,20 @@ internal abstract partial class VariableSymbol : Symbol, ITypedSymbol
     /// True, if this local is mutable.
     /// </summary>
     public abstract bool IsMutable { get; }
+
+    public override Api.Semantics.Visibility Visibility
+    {
+        get
+        {
+            var syntax = this.DeclaringSyntax as FunctionDeclarationSyntax;
```

Comment:
Function declaration syntax? I have a suspicion this is untested if this has sled through this far, please investigate

Reply:
Before fixing, make this break with a test!